core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -550,7 +565,7 @@ void collect( OSMWay way, double wayTypeSpeed, TreeMap<Double, Integer> weightTo
     {
         String service = way.getTag("service");
         String highway = way.getTag("highway");
-        if (way.hasTag("bicycle", "designated") || way.hasTag("bicycle", "official"))
+        if ((highway!=null) && (way.hasTag("bicycle", "designated") || way.hasTag("bicycle", "official")))
```

Comment:
I checked. You are right. As this does not make sense OSMI provides a check for it, so apperently this is accepted the OSM API -> I added that check.